workflows/flows/copy_amplicon_pipeline_results.py

```diff
@@ -0,0 +1,15 @@
+from prefect import flow
+
+from workflows.prefect_utils.slurm_flow import move_data
+
+
+@flow(
+    name="Copy Amplicon Pipeline Results",
+    flow_run_name="Copy Amplicon Pipeline Results",
+)
+async def copy_amplicon_pipeline_results():
```

Comment:
This should take a param of `analysis: analyses.models.Analysis`, because in the end we will call this `copy_amplicon_pipeline_results` flow as a subflow of the `analysis_amplicon_study` flow.

So, this flow can get the `move_data` source dir from `analysis.results_dir` and `EMG_CONFIG.slurm.default_workdir`.

The destination (FTP) should be something like `Path(EMG_CONFIG.slurm.ftp_results_dir) / analysis.study.first_accession[:4] / analysis.study.first_accession`